### PR TITLE
Fix integer division, see #4

### PR DESCRIPTION
**Pull request by agoscinski**
Solved issue #4. Please review @PicoCentauri

### CODE REVIEW

File: src/sample/simple.py [hunk 1]
```diff
@@ -7,4 +7,4 @@ def subtract_one(number):
 
 
 def inverse(number):
-    return 1 // number
+    return 1 / number
```
Comment (PicoCentauri) [inline src/sample/simple.py:15]: Could you maybe add a docstring to this function. This makes it easier for new users to apply it.

**Review by PicoCentauri — CHANGES_REQUESTED** [review]
Thanks for this awesome change!

Maybe add a test case for your change to avoid future issues.

File: src/sample/simple.py (hunk 1 above)
Comment (agoscinski) [reply src/sample/simple.py:15]: done

**Comment by agoscinski** [comment]
everything fine merge conflict solved

File: tests/test_simple.py [hunk 2]
```diff
@@ -21,6 +21,8 @@ def test_inverse(self):
     def test_inverse_0(self):
         with self.assertRaises(ZeroDivisionError):
             inverse(0)
+        with self.assertRaises(ZeroDivisionError):
+            inverse(1)
```
Comment (PicoCentauri) [inline tests/test_simple.py]: @agoscinski this test is not testing your changes. Are you able to adjust them?

**Review by PicoCentauri — CHANGES_REQUESTED** [review]

**Pull request closed without merging**